apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -320,6 +328,14 @@ export default class Simple2Sequencer extends Sequencer {
   }
 
   private addNewEvent<T extends PlaybackEvent>(event: T) {
+    this.currentEventCount++;
+    if (this.currentEventCount === MAX_NUMBER_EVENTS) {
+      console.log(`Reached MAX_NUMBER_EVENTS (${MAX_NUMBER_EVENTS}) events.`);
+    }
+    if (this.currentEventCount >= MAX_NUMBER_EVENTS) {
+      return;
+    }
```

Comment:
nit: could this be:
```
    if (this.currentEventCount >= MAX_NUMBER_EVENTS) {
      return;
    }
    this.currentEventCount++;
    if (this.currentEventCount === MAX_NUMBER_EVENTS) {
      console.log(`Reached MAX_NUMBER_EVENTS (${MAX_NUMBER_EVENTS}) events.`);
    }
```
to avoid incrementing once we pass the limit? Also do we want to log to cloudwatch for monitoring purposes or just keep this in browser logs for now? To be fair there's not much action for us to take if a user reaches this limit, so browser logs are probably sufficent.

Reply:
Oh good question.  Might be okay for now, but we should discuss for a potential follow-up.